### PR TITLE
fix: support Yarn alias dependencies

### PR DESCRIPTION
**Pull request by kyegupov**
- [x] Ready for review
- [x] Follows [CONTRIBUTING](https://github.com/snyk/snyk/blob/master/.github/CONTRIBUTING.md) rules
- [x] Reviewed by Snyk internal team

#### What does this PR do?

See README.md in the changed files

#### Where should the reviewer start?

README.md + deps.js

#### How should this be manually tested?

Build snyk-cli against it and test the case from the github issue (below)

#### What are the relevant tickets?

https://github.com/snyk/snyk/issues/330

### CODE REVIEW

File: test/fixtures/pkg-yarn-renamed-deps/README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+Both npm and Yarn support "GitHub URL" kind of dependency:
```
Comment (lili2311) [inline test/fixtures/pkg-yarn-renamed-deps/README.md:1]: ❤️ Love the README

File: test/fixtures/pkg-yarn-renamed-deps/node_modules/githubrenamed/.travis.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+language: node_js
```
Comment (lili2311) [inline test/fixtures/pkg-yarn-renamed-deps/node_modules/githubrenamed/.travis.yml]: is it needed? Maybe trim down the fixtures to what we need only?

File: test/fixtures/pkg-yarn-renamed-deps/node_modules/githubrenamed/LICENSE [hunk 3]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
```
Comment (lili2311) [inline test/fixtures/pkg-yarn-renamed-deps/node_modules/githubrenamed/LICENSE]: same here, let's ditch from the fixtures?

File: test/fixtures/pkg-yarn-renamed-deps/node_modules/githubrenamed/.travis.yml (hunk 2 above)
Comment (kyegupov) [reply test/fixtures/pkg-yarn-renamed-deps/node_modules/githubrenamed/.travis.yml]: I don't think this file is there anymore, how do you see it? :)

**Review by lili2311 — APPROVED** [review]

**Pull request merged**